members.json

```diff
@@ -1367,6 +1367,16 @@
 				"affiliation": "The News Corporation"
 			}
 		]
+	},
+	{
+		"name": "Anqi Li",
+		"term": [
+			{
+				"start": "2024-12-05",
+				"chair": "true",
+				"type": "staff-contact"
```

Comment:
```suggestion
				"type": "appointed"
```
Angel is from the team, but her role is not staff contact, so I think the above would be more in line with how non-elected chairs were described before.

Reply:
The Process does use the word "appointed" in reference to a chair not from the elected members:
> With the input of the [AB](https://www.w3.org/policies/process/#advisory-board), the [Team](https://www.w3.org/policies/process/#team) appoints the Chair, who should choose a co-chair among the elected participants
So that's good enough for me.